server/CLI/manager.py

```diff
@@ -0,0 +1,205 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Utility for controlling(creating/deleting/printing) the clients instances"""
+
+import asyncio
+import os
+import secrets
+import ssl
+import string
+import sys
+from argparse import ArgumentParser
+from email.message import EmailMessage
+from smtplib import (SMTP_SSL, SMTPHeloError, SMTPAuthenticationError,
+                     SMTPException)
+from asyncpg.exceptions import UniqueViolationError
+from dotenv import load_dotenv
+from huntflow_reloaded.models import User, gino_run  # pylint: disable=import-error
+
+
+load_dotenv()
+
+
+def send_mail(smtp_server, port, sender_email, sender_password, receiver_email,  # pylint: disable=redefined-outer-name,too-many-arguments
+              message):  # pylint: disable=redefined-outer-name
+    """Sending credentials to user. """
+    context = ssl.create_default_context()
+    with SMTP_SSL(smtp_server, port, context=context) as server:
+        server.login(sender_email, sender_password)
+        msg = EmailMessage()
+        msg['Subject'] = 'Your credentials'
+        msg['From'] = sender_email
+        msg['To'] = receiver_email
+        msg.set_content(message)
+        server.sendmail(sender_email, receiver_email, msg.as_string())
+
+
+def generate_password(pass_len):
+    """Creating a password of the specified length. """
+    chars_set = string.ascii_letters + string.digits
+    return ''.join(secrets.choice(chars_set) for _ in range(pass_len))
+
+
+async def create_user(email, password):  # pylint: disable=redefined-outer-name
+    """Creating the user instance. """
+    record = await User.create(
+        email=email,
+        password=password
+    )
+    return record
+
+
+async def delete_user(email):
+    """Deleting the user instance. """
+    user = await User.query.where(User.email == email).gino.all()
+    if not user:
+        return False
+    user = user[0]
+    await user.delete()
+    return True
+
+
+async def list_of_users():
+    """Printing the list of users. """
+    all_users = await User.query.gino.all()
+    return all_users
+
+
+def connect_to_postgres(loop, postgres_host, postgres_port, postgres_user,
+                        postgres_password):
+    """Connecting to postgres server. """
+    if not all([postgres_host, postgres_user, postgres_password,
+                postgres_port]):
+        sys.stderr.write('PostgreSQL server config not found!\n')
+        sys.exit(1)
+    connection_str = 'postgresql://{}:{}@{}:{}/gino'.format(postgres_user,
+                                                            postgres_password,
+                                                            postgres_host,
+                                                            postgres_port)
+    loop.run_until_complete(gino_run(connection_str))
```

Comment:
@polina-popova Anyway we need to call it inside event loop. So this line is necessary. Otherwise we get error RuntimeError "coroutine was never awaited"